doc/nrf/gsg_guides/nrf7002_gs.rst

```diff
@@ -286,28 +286,20 @@ This allows to build and run secure-only applications on the nRF5340 SoC.
 Building |NCS| applications with Arm TrustZone
 ==============================================
 
-Applications on nRF5340 can use Cortex-M Security Extensions (CMSE) and separate firmware for the application core between Secure Processing Environment (SPE) and Non-Secure Processing Environment (NSPE).
-You can build SPE using either |NCS| or `Trusted Firmware M`_ (TF-M).
-You must always build NSPE using |NCS|.
+Applications on nRF5340 can use the M33 TrustZone for :ref:`security by separation <ug_tfm_security_by_separation>`.
+When security by separation is used, the MCU always starts executing from the secure area at boot.
+The secure bootloader chain starts the :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`, which configures a part of memory and peripherals to be non-secure, and then jumps to the user application located in the non-secure area.
```

Comment:
You can technically also run TF-M without bootloaders. It is not practical but as a simpler configuration that we use in the samples for example. So maybe here you can say (when enabled, the secure bootloader chain ....."